src/collection-browser.ts

```diff
@@ -376,6 +381,8 @@ export class CollectionBrowser
         .displayMode=${this.displayMode}
         .selectedTitleFilter=${this.selectedTitleFilter}
         .selectedCreatorFilter=${this.selectedCreatorFilter}
+        .titleLetterCounts=${this.titleLetterCounts}
```

Comment:
Note: reviewing to the end, it seems like the Title letter picker UI and Creator letter picker UI already use separate `titleLetterCounts` and `creatorLetterCounts` -- but I haven't looked at the UI. 

Maybe combining these has implications for existing UI code -- and my comments are, if not misplaced, than are not helpful because of constraints of existing code.